src/types.md

```diff
@@ -12,8 +12,8 @@ limited capabilities.
 The list of types is:
 
 * Primitive types:
-    * [Boolean] — `true` or `false`
-    * [Numeric] — integer and float
+    * [Boolean] — `bool`
+    * [Numeric] — `integer` and `float`
```

Comment:
I would prefer to not use code spans for integer and float. The others are code spans because they are literal identifiers that can be used in code. "integer" and "float" are just words to encompass the types that are numeric.